### PR TITLE
fix icon shrink and use avatar for logo and icons

### PR DESCRIPTION
**Pull request by ehconitin**
- Fixed icon shrinking on tabs. The introduction of EllipsisDisplay takes 100% of the width, thus shrinking the icons.
- Removed scroll wrapper tablist. This was removed in #9016 but reintroduced in #9089. This reintroduction made the dark border below the active tab disappear. 
- Used Avatar for icon and logo rendering following the changes made in #9093

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/tab/components/Tab.tsx [hunk 1]
```diff
@@ -92,8 +96,24 @@ export const Tab = ({
...
+              size="md"
+              placeholder=""
+              iconColor={iconColor}
+            />
+          )}
+          {Icon && (
+            <Avatar
+              Icon={Icon}
+              size="md"
+              placeholder=""
```
Comment (ijreilly) [inline packages/twenty-front/src/modules/ui/layout/tab/components/Tab.tsx]: from Avatar definition

```
const placeholderChar = placeholder?.[0]?.toLocaleUpperCase();
  const showPlaceholder =
    noAvatarUrl || invalidAvatarUrls.includes(avatarImageURI);
(...)
{Icon ? (
        <Icon
          color={iconColor ? iconColor : 'currentColor'}
          size={theme.icon.size.xl}
        />
      ) : showPlaceholder ? (
        placeholderChar
      ) : (
        <StyledImage src={avatarImageURI} onError={handleImageError} alt="" />
      )}
```

I fear we end up dispaying 'undefined' ?

Comment (ehconitin) [reply packages/twenty-front/src/modules/ui/layout/tab/components/Tab.tsx]: fixed! realized it after grep's review :)

**Comment by charlesBochet** [comment]
@ehconitin why do we remove the scrollwrapper in tab list? we want it in case we have too many tabs and need horizontal scrolling isn't it?

**Comment by ehconitin** [comment]
I reverted the tabList changes from #8726 and wrapped the tabList in WorkflowEditActionFormServerlessFunction in a container(as done elsewhere in the codebase).
Added scrollWrapper back similar to #9089!

**Comment by charlesBochet** [comment]
That's a mistake, we still need the scrollWrapper!

**Comment by ehconitin** [comment]
@charlesBochet added it back! ready for review!

**Review by charlesBochet — APPROVED** [review]

**Comment by charlesBochet** [comment]
Discussed this one with @Bonapara to understand the change:
- The current Avatar component that is able to take a logo, an icon, should be renamed to Icon
- we should introduce an other sub-component called IconAvatar that should only take logo or placeholder into props
- current Icon should become IconTablerIcon
- we will introduce other types of icons later such as: IconColor, IconFilledIcon, IconEmoji

**Pull request merged**